Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM golang:1.21-alpine as base
-MAINTAINER team@pganalyze.com
```

Comment:
For other reviewers, this was [superseded by LABEL](https://docs.docker.com/reference/dockerfile/#maintainer-deprecated).

Reply:
Thanks for the note! Yeah I added `label` a while back so it's safe to remove/retire: https://github.com/pganalyze/collector/pull/534/files#r1571635098